src/plugins/bookPlayer/tableOfContents.js

```diff
@@ -51,6 +51,25 @@ export default class TableOfContents {
         });
     }
 
+    chapterTocItem(book, chapter) {
+        let itemHtml = '<li>';
+
+        // remove parent directory reference from href to fix certain books
+        const link = chapter.href.startsWith('../') ? chapter.href.slice(3) : chapter.href;
+        itemHtml += `<a href="${book.path.directory + link}">${chapter.label}</a>`;
```

Comment:
There should be html escaping here to prevent xss.

You will need to add the import `import escapeHTML from 'escape-html';` and then escape the link and label:

```suggestion
        itemHtml += `<a href="${escapeHTML(book.path.directory + link)}">${escapeHTML(chapter.label)}</a>`;
```